tests/note_test.rs

```diff
@@ -175,8 +172,9 @@ fn note_keys_consistency() {
     assert!(vk.owns(&note));
 }
 
+#[should_panic]
 #[test]
-fn fee_and_crossover_generation() -> Result<(), Error> {
```

Comment:
any specific reason you removed the `Result` return? If you leave it there you can use the `?` a few lines down instead of unwrapping